app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentExecutionLabel.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import {OverlayTrigger, Tooltip} from 'react-bootstrap';
+
+import {ManualJudgmentApproval} from './ManualJudgmentApproval';
+import {IExecutionStageSummary} from 'core/domain/index';
+import {IExecution} from 'core/domain/IExecution';
+import {Application} from 'core/application/application.model';
+import {HoverablePopover} from 'core/presentation/HoverablePopover';
+import {ExecutionBarLabel} from 'core/pipeline/config/stages/core/ExecutionBarLabel';
+
+interface IState {
+  target?: any;
+}
+
+interface IProps {
```

Comment:
*Super* pedantic but since props comes before state in all functions (and logically as the component gets constructed, I like the `IProps` interface before the `IState` interface.